source

```diff
@@ -14,7 +14,7 @@ if not game:IsLoaded() then
 	notLoaded:Destroy()
 end
 
-currentVersion = '5.9.3'
+currentVersion = '6.0.0'
```

Comment:
i changed the version number too. this should really be an actual github package release and not hard-coded at the top of the script but we'll deal with that later.